### DISCUSSION TITLE
Horizontal Timeline

### DISCUSSION THREAD

**Discussion by devklick** [mantinedev/mantine#5978] [Feature requests]
The [`Timeline` component](https://mantine.dev/core/timeline/) is a nice feature, however I feel that it would be beneficial to support horizontal timelines as well as vertical timelines. Having only vertical timelines feels somewhat limiting when designing the UI.

Perhaps adding a `direction: 'horizontal' | 'vertical'` prop would be suitable, of course defaulting to `vertical` to prevent breaking changes.

**Comment by rtivital** [comment]
See Stepper component

**Comment by devklick** [comment]
Amazing, thank you! :grinning:
